src/Powercord/plugins/pc-moduleManager/index.js

```diff
@@ -279,10 +279,17 @@ module.exports = class ModuleManager extends Plugin {
     this._quickCSS = css.trim();
     this._quickCSSElement.innerHTML = this._quickCSS;
     if (save) {
-      await writeFile(this._quickCSSFile, this._quickCSS);
+      await this._saveQuickCSS(null, false);
     }
   }
 
+  async _saveQuickCSS (css, apply = true) {
+    if (apply) {
+      await this._applyQuickCSS(css, false);
+    }
+    await writeFile(this._quickCSSFile, this._quickCSS);
```

Comment:
```js
    this._quickCSS = css.trim();
    this._quickCSSElement.innerHTML = this._quickCSS;
```
add these two as well and put `css` in line 282 function as well

Reply:
why should i add them if they are already in `_applyQuickCSS`